README.md

```diff
@@ -1,6 +1,6 @@
 # Python tools for Visual Studio
 
-Python support for [Visual Studio 2017](https://aka.ms/PTVS) is developed and managed here.
+Python support for [Visual Studio](https://aka.ms/PTVS) is developed and managed here.
 
 Visit our page on [VisualStudio.com](https://aka.ms/PTVS) for an overview and download links, and check out our documentation on [docs.microsoft.com](https://aka.ms/PTVSDocs) and our (old) [Getting Started](http://aka.ms/PTVSTutorial) series on YouTube.
```

Comment:
The "aka.ms/PTVS" link on this line and above are the same line which duplicates information for no benefit. Please modify either one of them. 

Also, add the links in line rather than describing and linking separately. For example: "You can visit our overview page (link), download links (link), documentation (link), and (old) YouTube tutorials (link).